src/components/terms-and-conditions.tsx

```diff
@@ -0,0 +1,205 @@
+// Terms and Conditions Pop-up
+
+import React, { ReactNode, useState } from "react";
+import { Inter } from "next/font/google";
+import { Controller, SubmitHandler, useForm } from "react-hook-form";
+
+const inter = Inter({ subsets: ["latin"] });
+
+type FormValues = {
+  tncAccepted: false;
+};
+
+const Terms_And_Conditions = () => {
```

Comment:
components should be declared using PascalCase. Please have a look at the style guide for reference